src/pages/en/guides/markdown-content.md

```diff
@@ -206,10 +206,31 @@ See the migration guide for help [converting your existing Astro `.md` files to
 
 ### Using Components in Markdown
 
-Please install the official [`@astrojs/mdx`](/en/guides/integrations-guide/mdx/) integration to continue to use Astro components or [UI framework components in MDX (`.mdx`) files](/en/core-concepts/framework-components/#using-framework-components).
+Please install the official [`@astrojs/mdx`](/en/guides/integrations-guide/mdx/) integration to continue to use [Astro components](/en/core-concepts/astro-components/) or [UI framework components](/en/core-concepts/framework-components/) in MDX (`.mdx`) files.
 
 See the migration guide for help [converting your existing Astro `.md` files to `.mdx`](/en/migrate/#converting-existing-md-files-to-mdx).
 
+
+## MDX
+
+Astro includes full support for MDX with the [official `@astrojs/mdx` integration](/en/guides/integrations-guide/mdx/). 
+
+Any `.mdx` page located in your project within `src/pages` will automatically generate a page route. 
+
+📚 Read more about using MDX in Astro in our [MDX integration guide](/en/guides/integrations-guide/mdx/).
+
+### Using Variables in MDX
+
+With the `@astrojs/mdx` integration, you can use [variables and JSX expressions in MDX (`.mdx`) files](/en/guides/integrations-guide/mdx/#variables).
+
+
+### Using Components in MDX
+
+With the `@astrojs/mdx` integration, you can use [Astro components](/en/core-concepts/astro-components/) or [UI framework components](/en/core-concepts/framework-components/) in MDX (`.mdx`) files just as you would [use them in any other Astro component](/en/core-concepts/framework-components/#using-framework-components).
```

Comment:
I feel like we may be hitting the threshold of link usefulness here especially given the last link is to the same page as the previous one. What do you think of unlinking that one?

```suggestion
With the `@astrojs/mdx` integration, you can use [Astro components](/en/core-concepts/astro-components/) or [UI framework components](/en/core-concepts/framework-components/) in MDX (`.mdx`) files just as you would use them in any other Astro component.
```

Reply:
You don't like all my links??  ;)

What about the opposite, what if I unlink generic components above, and ONLY leave this link, because it's very explicit about HOW to use the components?